### PR TITLE
chore: Update downstream repo for recomm updates due to monorepo

### PR DESCRIPTION
**Pull request by Weetbix**
Updates the repo and workflow for recomm api definition updates based on the new monorepo.

### CODE REVIEW

**Review by mehicalen — APPROVED** [review]

**Comment by Weetbix** [comment]
@php please review

File: .github/workflows/trigger-downstream-updates.yaml [hunk 1]
```diff
@@ -51,9 +51,9 @@ jobs:
       - name: Trigger update workflow
         uses: benc-uk/workflow-dispatch@v1
         with:
-          workflow: Update API definitions
+          workflow: FE/Recomm/Update resources from API definitions
```
Comment (ashkarpetin) [inline .github/workflows/trigger-downstream-updates.yaml:54]:
```suggestion
          workflow: Update FE resources from API definitions
```

Comment (Weetbix) [reply .github/workflows/trigger-downstream-updates.yaml:54]: This needs to match the name of the workflow we are trying to trigger: https://github.com/Rebilly/rebilly/blob/main/.github/workflows/fe.recomm.update-resources.yaml#L1

**Comment by Weetbix** [comment]
php please review

**Review by lexerom — APPROVED** [review]

File: .github/workflows/trigger-downstream-updates.yaml (hunk 1 above)
Comment (ashkarpetin) [reply .github/workflows/trigger-downstream-updates.yaml:54]: `RE/Recomm/Update resources from API definitions` make doesn't make  much sense to me. What does is mean?

RE or Recomm or Update  resources from API definitions?

**Review by ashkarpetin — APPROVED** [review]

**Pull request merged**